lib/taurus/core/tango/tangofactory.py

```diff
@@ -182,6 +183,16 @@ def get_default_tango_host(self):
         """Retruns the current default tango host
         """
         return self._default_tango_host
+    
+    def set_tango_events_disabled(self,value):
```

Comment:
IMHO, it would be better if the property was named tango_events_enabled:

set_tango_events_enabled
is_tango_events_enabled